README.md

```diff
@@ -1,24 +1,150 @@
+
+* Clicking on each category tab will allow you to use the interface to annotated the values of each categorized column.
+* When multiple columns have been linked with a category, each column will have its own column in the annotation components on the page.
+* Any value in the annotation interface can be marked as a 'missing value' (i.e. `'N/A'`, empty string, etc.)
+
+![Home page with uploaded table and dictionary files](./images/readme_annotation_page.png)
+
+#### Age rows (continuous values)
+
+![Home page with uploaded table and dictionary files](./images/readme_annotation_page_continuous_value.png)
+
+All values for columns categorized as `Age` can be annotated here with a set of continuous value transformations. Clicking on the dropdown will allow you to select a transformation. Current transformations available include `bounded`, `euro`, `float`, `int`, and `iso8601`. The raw values from the column are shown and when a transformation type is selected a preview of how that transformation would alter each raw value is shown.
+
+#### Sex and Diagnosis rows (categorical values)
+
+All values for columns categorized as either `Sex` or `Diagnosis` can be annotated here with labels that come from Neurobagel and the controlled vocabularies it utilizes to help form the Neurobagel metadata model. Clicking on the drop down on each raw value row will allow you to select an appropriate Neurobagel label for this value.
+
+![Home page with uploaded table and dictionary files](./images/readme_annotation_page_categorical_value.png)
+
+#### Missing values
+
+Any continuous or categorical value can be marked as 'missing' via the adjacent `Mark as missing` button. This will remove the value from the annotatable values and place it in the `Missing values` section on the page. This can be undone for any value by clicking the `Not Missing` button in this section of the page.
+
+![Home page with uploaded table and dictionary files](./images/readme_annotation_page_missing_values.png)
+
+#### Unlinking columns
+
+Data table columns can also be unlinked (e.g. un-categorized) on the Annotation page in this section of the page labeled `Review the annotated columns`. This will stop the column from being annotated and have any annotations made for its values removed. The change is also reflected on the previous Categorization page.
+
+![Home page with uploaded table and dictionary files](./images/readme_annotation_page_unlinking.png)
+
+_Next page criteria:_ At least one annotation must be made from any of the categorized columns
+
+### Download page
+
+Click the <span style="color:green;">download annotated data button</span> to download what we refer to an 'annotated' data dictionary that is a Neurobagel-enhanced BIDS-style data dictionary. This file will include any entries in the original data dictionary that you uploaded on the home page.
+
+![Home page with uploaded table and dictionary files](./images/readme_download_button.png)
+
+## Developer information
```

Comment:
I think it's better to have this section under `Setup` since most folks who just wanna get started may setup and not read enough into the instructions to see the warning.